content/influxdb/v2.0/monitor-alert/notification-endpoints/create.md

```diff
@@ -26,7 +26,7 @@ To send notifications about changes in your data, start by creating a notificati
 5.  In the **Name** and **Description** fields, enter a name and description for the endpoint.
 6.  Enter information to connect to the endpoint:
 
-    - For HTTP, enter the **URL** to send the notification. Select the **auth method** to use: **None** for no authentication. To authenticate with a username and password, select **Basic** and then enter credentials in the **Username** and **Password** fields. To authenticate with a token, select **Bearer**, and then enter the authentication token in the **Token** field.
+    - For HTTP, enter the **URL** to send the notification. Select the **auth method** to use: **None** for no authentication. To authenticate with a username and password, select **Basic** and then enter credentials in the **Username** and **Password** fields. To authenticate with a token, select **Bearer**, and then enter the API token in the **Token** field.
```

Comment:
This one is referring to the token that gets sent with the `Authorization` header. We can probably just remove authentication/API.

```suggestion
    - For HTTP, enter the **URL** to send the notification. Select the **auth method** to use: **None** for no authentication. To authenticate with a username and password, select **Basic** and then enter credentials in the **Username** and **Password** fields. To authenticate with a token, select **Bearer**, and then enter the token in the **Token** field.
```